third_party/terraform/utils/utils.go

```diff
@@ -350,8 +350,14 @@ func retryTimeDuration(retryFunc func() error, duration time.Duration) error {
 }
 
 func isRetryableError(err error) bool {
-	// 409's are retried because cloud sql throws a 409 when concurrent calls are made
 	if gerr, ok := err.(*googleapi.Error); ok && (gerr.Code == 409 || gerr.Code == 429 || gerr.Code == 500 || gerr.Code == 502 || gerr.Code == 503) {
+		if gerr.Code == 409 && !strings.Contains(gerr.Body, "operationInProgress") {
```

Comment:
nit: Should this be broken out to a separate block? Needing to guard against nearly every `409` type makes me feel like we should have a block `if gerr.Code == 409 && strings.Contains(gerr.Body, "operationInProgress"`.